tests/math/test_functions.py

```diff
@@ -607,6 +627,11 @@ def test_numpy(self, t):
         """Vanilla NumPy arrays, sequences, and lists will always return False"""
         assert not fn.requires_grad(t)
 
+    @pytest.mark.parametrize("t", [jnp.array([1, 2, 3])])
+    def test_jax(self, t):
+        """Vanilla NumPy arrays, sequences, and lists will always return False"""
+        assert fn.requires_grad(t)

```

Comment:
Question: should we instead have them always return `True`?

Although I suppose neither is correct since JAX is functional wrt how differentiation takes place.

Reply:
It actually does return `True`, I just need to fix the docstring.